### PR TITLE
Feature/remove flag as language

### PR DESCRIPTION
**Pull request by mfattoru**
The language now is displayed with the local translation of the language

screenshot-20170705095318-1148x246

### CODE REVIEW

File: src/pretix/presale/context.py [hunk 1]
```diff
@@ -53,6 +54,8 @@ def contextprocessor(request):
             ctx['css_file'] = default_storage.url(request.event.settings.presale_css_file)
         ctx['event_logo'] = request.event.settings.get('logo_image', as_type=str, default='')[7:]
         ctx['event'] = request.event
+        for code in request.event.settings.locales:
+            ctx.setdefault('languages',[]).append( get_language_info(code) )
```
Comment (raphaelm) [inline src/pretix/presale/context.py]: Hmm, I'm not really a fan of calling ``setdefault`` that many times, although it probably doesn't matter. I think a list comprehension would be cleaner:
```
ctx['languages'] = [
    get_language_info(code) for code in request.event.settings.locales
]
```
What do you think?

Comment (mfattoru) [inline src/pretix/presale/context.py]: yes, your style looks cleaner than mine

**Comment by raphaelm** [comment]
Wonderful! Congratulations on your very first pull request and thank you very much! :sparkles:

**Comment by mfattoru** [comment]
Thank you for assisting me in my first pull request,it really put me in the mood of contributing to more projects. Didn't expect to find friendly and helpful people assisting.
Thank you so much

**Pull request merged**